src/App.tsx

```diff
@@ -1,61 +1,114 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+
+function findUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId);
+}
+
+const todos = todosFromServer.map(todo => {
+  const user = findUserById(todo.userId);
+
+  return { ...todo, user };
+});
 
 export const App = () => {
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [title, setTitle] = useState('');
+  const [updatedTodos, setUpdatedTodos] = useState<Todo[]>(todos);
+  const [titleError, setTitleError] = useState(false);
```

Comment:
Let me be a nerd, as Vitalii Schukin  would say)) It's better to insert 'is' or 'has' to the title of the state if it is a boolean

Reply:
Good point, fix this